### PR TITLE
Add minimal Lambda app, update docker-compose

### LINKED ISSUE #351: Create a minimal Lambda backend
**Background**

The new tech stack uses Lambda on the backend. We should make a minimal API on Lambda that can respond with the JSON "ok" on one endpoint. Since our Lambda application will be deployed on AWS, this will need resources from #349 as well as deployment tooling from #347.

**We'll know we're done when...**
- [ ] There is an absolutely minimal API running in our development environment

**Tasks**
- [ ] Research and decide on how our backend application will get deployed by CI/CD
- [ ] Write the world's smallest API on Lambda

### PR DESCRIPTION
**Pull request by beechnut**
- Adds a minimal Lambda API that can be run locally and deployed via Claudia.js.
- Changes `api` service in Docker Compose config to use the Lambda API.

Closes #351

### CODE REVIEW

File: api/bin/index.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+"use strict";
```
Comment (neilmb) [inline api/bin/index.js]: I wonder about versioning the generated JS files from our TypeScript? I think that I would usually not version control compiled versions of other source files. Should we `.gitignore` `api/bin/*.js` too?

File: api/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import express, { Request, Response, NextFunction } from 'express';
+
+const app  = express();
+const port = 3000;
+
+app.listen(port, () => {
+  console.log(`Timezones by location app is running on port ${port}.`)
+});
+
+interface LocationWithTimezone {
```
Comment (neilmb) [inline api/src/index.ts:10]: ❤️  strong typing

**Review by neilmb — CHANGES_REQUESTED** [review]
Besides versioning the generated files in `bin/`, this looks great.

File: api/bin/index.js (hunk 1 above)
Comment (beechnut) [reply api/bin/index.js]: Good catch! I'll make that change now.

**Review by neilmb — APPROVED** [review]

**Pull request merged**